EpiAware/src/EpiInfModels/ODEProcess.jl

```diff
@@ -0,0 +1,156 @@
+@doc raw"""
+A structure to hold the initial condition and parameters for an ODE (Ordinary Differential Equation) process.
+`params::ODEParams` is used in the method `generate_latent_infs(epi_model::ODEProcess, params::ODEParams)`
+
+# Constructors
+- `ODEParams(; u0::VecOrMat, p::VecOrMat)`: Create an `ODEParams` object with the initial condition(s) `u0` and parameters `p`.
+
+# Example
+```jldoctest
+using EpiAware
+params = ODEParams(; u0 = ones(10), p = [2, 3])
+
+# output
+
+ODEParams{Float64}([1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0], [2.0, 3.0])
+```
+"""
+struct ODEParams{T}
+    "The initial condition(s) for the ODE, which can be a vector or matrix of type `T`."
+    u0::VecOrMat{T}
```

Comment:
Can you clarify why `VecOrMat isn't in the {} definition?

Reply:
To me this suggests a much more flexible typing approach than we have used elsewhere